python_ta/reporters/html_server.py

```diff
@@ -0,0 +1,57 @@
+import os
+import sys
+import webbrowser
+from http.server import BaseHTTPRequestHandler, HTTPServer
+
+
+def open_html_in_browser(html: bytes, watch: bool, port: int) -> None:
+    """
+    Display html in a web browser without creating a temp file.
+    Instantiates a trivial http server and uses the webbrowser module to
+    open a URL to retrieve html from that server.
+    If watch is False, the server responds to exactly one request and then shuts down.
+    If watch is True, the server runs indefinitely, allowing multiple requests.
+    Adapted from: https://github.com/plotly/plotly.py/blob/master/packages/python/plotly/plotly/io/_base_renderers.py#L655
+    """
+
+    class RequestHandler(BaseHTTPRequestHandler):
+        def do_GET(self):
+            self.send_response(200)
+            self.send_header("Content-type", "text/html")
+            self.end_headers()
+
+            buffer_size = 1024 * 1024
+            for i in range(0, len(html), buffer_size):
+                self.wfile.write(html[i : i + buffer_size])
+
+        def log_message(self, format, *args):
+            """Overridden so that no server logging is printed."""
+            pass
+
+    if watch:
+        print(
+            "[INFO] Your PythonTA report is being opened in your web browser.\n"
+            "       Press Ctrl + C, or stop this program to exit.\n\n",
```

Comment:
Delete the comma on this line. Also remove the two ending `\n`, they aren't necessary